Sources/Scheduler.swift

```diff
@@ -113,10 +122,21 @@ public final class UIScheduler: SchedulerProtocol {
 				action()
 			}
 
-			OSAtomicDecrement32(&self.queueLength)
+			#if os(Linux)
```

Comment:
I think these implementation details shouldn't be here. Maybe we can make an `internal class IntAtomic: AtomicProtocol`, so that this class works evenly regardless of the actual type being used?